packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -95,6 +95,14 @@ export default class Extension {
     return { exportedJson: JSON.stringify(keyring.backupAccount(keyring.getPair(address), password)) };
   }
 
+  private async accountsBatchExport ({ addresses }: RequestAccountBatchExport): Promise<ResponseAccountExport> {
```

Comment:
Needs the password to encrypt the JSON export. (password-less it will be encoded with no encryption)

Reply:
ok, so this password is used to encrypt all accounts exported right? and we don't need to verify it.
I'll need to open another PR in `ui` repo.

Reply:
I find it a bit hard to design UI for entering password before exporting all.
Got any idea?

Reply:
The UI is rough, but the function solid.

![1](https://user-images.githubusercontent.com/17879773/110821525-21161780-82cb-11eb-982a-e688e48d0286.gif)